Assets/Scripts/ScriptingSystem/GameManager.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    private class nodeHolder
+    {
+        public Head head;
+        public Node currentNode;
+
+        public nodeHolder(Head head)
+        {
+            this.head = head;
+            currentNode = head;
+        }
+    }
+
+    List<nodeHolder> nodeHolders = new List<nodeHolder>();
+    private bool paused = true;
+    private float time = 0;
+    [SerializeField] private float TickTimeInSeconds = 1;
+    private int tick = 0;
```

Comment:
Time & Tick have 0 hard defined, this is not necesairy since the compiler will set this as default.